### ISSUE TITLE
Login “Forgot password” was skipped in keyboard tab flow

### ISSUE THREAD

**Issue by mgifford** [primeroIMS/primero#500]
I wanted to explore this application and and its accessibility. I got it set up and just wanted to report a few issues.

Problem: The control behaved like a link button but was not consistently keyboard-focusable.

Discovered: Manual keyboard testing on login screen.

How to replicate: Open login page, use Tab from username/password/login; focus could skip the forgot-password control.

Standards impacted: WCAG 2.1.1 Keyboard, WCAG 2.4.3 Focus Order.

Upstream location: [upstream link component](https://github.com/primeroIMS/primero/blob/develop/app/javascript/components/action-button/components/link/component.jsx)
Your change: [fork commit e66900613f](https://github.com/mgifford/primero/commit/e66900613f#diff-d293fc6fca33da599c81d05579efacd4be51a0025733fa80e2cdf2298abee032R14-R19) and file [fork link component](https://github.com/mgifford/primero/blob/develop/app/javascript/components/action-button/components/link/component.jsx)

I'm sorry that there's some other cruft in this. I should have broken this into smaller PRs.
